src/controller/user.controller.ts

```diff
@@ -1,5 +1,5 @@
 import { Request, Response, NextFunction } from 'express';
-import { insertUser, getUserById, getUserDetails, updateUserbyId, checkUserByUsernameAndPassword } from '../service/user.service';
+import { insertUser, getUserById, getUserDetails, updateUserbyId, checkUserByUsernameAndPassword, getUserMediumPosts } from '../service/user.service';
```

Comment:
@rustmn brother, these functions should be in a separate file.
for example, this is for getting Medium posts. So,  it should in a separate file called `medium.controller.ts`